Update ru_ru to 3.1.1

Tefnya opened the pull request:
### Contributor Liscence Agreement
[Full CLA](https://github.com/iron431/Irons-Spells-n-Spellbooks/blob/1.19.2/CLA.md)
*Contributors: Please "X" in the following box to acknowledge our CLA*
- [X] I have read and agree to the CLA for Iron's Spells and Spellbooks which allows me to retain my ownership in the code submitted while granting the repository owner legal rights to use my contribution.

Tefnya committed:
Update ru_ru.json

iron431 reviewed (changes requested):
It seems that formatting has been removed from various tooltip components (such as tooltip.irons_spellbooks.enhance_spell_level) and that various old translations have been changed, some of which (such as spell descriptions) seem to less accurately depict the english version. can you explain why these changes were made?

Tefnya committed:
ru_ru slight improvements

Tefnya commented:
What do you mean removed? I only removed space as you did in en_us for tooltip.irons_spellbooks.enhance_spell_level. If you mean specifically line 154 I returned it, checked every other line and all spaces are placed according to en_us.

As for changes i made them for 3 reasons:
1) Basically the main thing i did is changing every "You" referring to player word to "Caster" (and all corresponding and dependent words), cause it felt more suitable for spell descriptions. I left it as it was only in descriptions of 2 weapon-based attacks and lightning lance for clarity.
2) Second change was deleting some verbs from beginning like "Conjure", "Manifest", "Cast", "Fire". I thought it doesn't change the meaning in any way and makes descriptions a bit more clear and easy to comprehend. I left actions only in places where they cannot be removed without affecting the meaning. I mean everyone knows what is beam and arrow, so why drawing actions into descriptions. But i see your point and i can change this one back if you think it would be more appropriate.
3) When playing around with the mod i noticed that a couple of spells (i remembered only one here) have slight difference in description and in-game mechanics (like wisp in line 499), so i corrected those in russian. For quick reference: it does not float to nearby entities, only to the targeted one. And if the entity is killed it disappears.

For now i changed few of them to be more precise in second commit. I as much as you want this translation to turn out good, so tell me if I need to revert second point (verbs in descriptions) back to previous translation variant👍.

Also a side note: en_us is probably missing "to" in line 535 after "damage"

Tefnya committed:
Add eldritch attributes (ru_ru.json)
